azure-iot-device/azure/iot/device/common/transport/mqtt/mqtt_topic_helper.py

```diff
@@ -0,0 +1,171 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
Mixed feelings about this module.

It definitely cleans up the code, so that's nice, but I'm not sure all of this should be maintained in the same place?

I'm good with it for now though.